File: Lista_2/Exercicios_SALA-lista-2/EX1-3Notas-s/EX1-s.c
#include <stdio.h>

int main() {
    float n1, n2, n3;
    scanf("%f", &n1);
    scanf("%f", &n2);
    scanf("%f", &n3);
    
    if ((n1 + n2 + n3) / 3 >= 6){
        printf("O aluno foi Aprovado com a média %.2f\n",(n1 + n2 + n3) / 3);
    }
    
    else{
        printf("O aluno foi Reprovado com a média %.2f\n",(n1 + n2 + n3) / 3);
    }
    return 0;
}

File: 01_IFSP/LP1/Lista_2/Exercicios_SALA-lista-2/EX1-3Notas-s/EX1-s.py
n1 = float(input())
n2 = float(input())
n3 = float(input())
situacao = "reprovado"
if (n1 + n2 + n3) / 3 >= 6:
    situacao = "aprovado"
print(f"O aluno foi {situacao} com a média {(n1 + n2 + n3) / 3:.2f}")